recipe/meta.yaml

```diff
@@ -19,13 +19,13 @@ requirements:
     - cmake
     - gmp
     - mpfr
-    - boost 1.62.*
+    - boost 1.63.*
```

Comment:
This can become `boost-cpp` after https://github.com/conda-forge/boost-feedstock/pull/27